From: methanal <baclofen@tuta.io>
Date: Fri, 8 Sep 2023 11:26:21 +0000
Subject: arm64: dts: qcom: sdm845-oneplus-common: add PDT override values

Provide the values of the function number and interrupt status
registers for partially implemented PDT entries of some aftermarket
touch ICs.

Signed-off-by: methanal <baclofen@tuta.io>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 5fb3f9a..29bce06 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -424,6 +424,8 @@ synaptics-rmi4-i2c@20 {
 		syna,reset-delay-ms = <200>;
 		syna,startup-delay-ms = <200>;
 
+		syna,pdt-desc = [34 41 01 01 12 01];
+
 		rmi4-f01@1 {
 			reg = <0x01>;
 			syna,nosleep-mode = <1>;
